tests/test_integrals.py

```diff
@@ -1620,11 +1618,29 @@ def test_bounce2d_checks(self):
             rtol=1e-2,
         )
 
-        # 10. Plotting
         fig, ax = bounce.plot(l, m, pitch_inv[l], include_legend=False, show=False)
         _, _ = bounce.plot_theta(l, m, show=False)
 
-        # make sure tests pass when spline=True
+        # check for consistency with different options
+        num_nufft = bounce.integrate(
+            integrand=TestBounce._example_numerator,
+            pitch_inv=pitch_inv,
+            data=_data,
+            points=points,
+            # ~1% of the integrals differ significantly at lower epsilon.
```

Comment:
In a few years time, I suspect finufft will fix some obscure bug (like the one they fixed around June this year in v.2.4.0) and this issue will vanish.